src/docs/learn/architecture/test.md

```diff
@@ -0,0 +1,147 @@
+# Test Infrastructure
+
+::: info
+
+This article serves as an invitation for sharing ideas to improve our test infrastructure,
+feel free to contact me on [twitter](https://x.com/boshen_c) or [discord](https://discord.gg/9uXCAwqQZW).
```

Comment:
Feel free to throw my name on here if you want. I'm around to help with support.

Reply:
changed to only discord.